src/developer/src/components/page/CreateKey.jsx

```diff
@@ -105,7 +105,9 @@ export function Component() {
     onSuccess: async () => {
       await queryClient.cancelQueries();
 
-      queryClient.invalidateQueries({ queryKey: ['my keys'] });
```

Comment:
**fyi**: this invalidates all queries that start with 'my keys'

Reply:
Oh? I was not seeing my newly created key when I went to the keys page immediately after creating it. Maybe I wasn't waiting long enough for it to re-query. Do you want me to change it back?